### ISSUE TITLE
Does not work on 22000.65

### ISSUE THREAD

**Issue by mdio33** [Konnor88/TenStartMenuFixer#1]
This patch does  not work on version 22000.65

**Comment by Konnor88** [comment]
Hi, I found the issue preventing it from working. It appears Microsoft has removed the Windows 10 start menu functionality in the latest Insider build. I have found a workaround, though it requires replacing the Windows 11 Build 22000.65 explorer with the explorer from the leaked build. I am planning on creating a tutorial on how to do that later today with the explorer.exe from the leaked build as a separate download. Thanks for notifying me about the issue.

UPDATE: It turns out my testing VM was on Insider Build 22000.51. I updated it to 22000.65 and the workaround doesn't appear to be working. I will be looking into more workarounds and give an update on whether or not if I find any.
